Segv in #612 is still an issue

QIvan opened the pull request:
that's because io.aeron.driver.MediaDriver.Context#close unmaps cnc and lossReoprt buffers.
But the media driver tries to write some counter value to the cnc.

QIvan commented:
> it does feel like you get what you deserve if you do this. ;-)

In fact I aggree with you. That was just my duty to report about such way to crush the Aeron which I met in our code. Anyway feel free to remove the code and close the issue.

> BTW this does not build. You need to run the build before submitting

oh really?... I'm awful sorry but I always run a buld before a push. Maybe I miss somethig this time because on my machine some of cluster tests fail... 
Honestly speaking I'm not fully understand the error

```
The command "eval git fetch origin +refs/pull/614/merge: " failed. Retrying, 2 of 3.
```

mjpt777 commented:
The build failed due to checkstyle. The indent level was 8 in your PR but the project rules are for 4. I merged before the build Travis CI build completed and fixed it.

QIvan commented:
> The build failed due to checkstyle. The indent level was 8 in your PR but the project rules are for 4. I merged before the build Travis CI build completed and fixed it.

oops, i'm sorry... That's strange because i fixed this indentation, may be it's some IDE trick. Anyway I'll be careful next time. 
Thanks!
